src/sentry/api/helpers/actionable_items_helper.py

```diff
@@ -0,0 +1,120 @@
+from sentry.models import EventError, PromptsActivity, SourceMapProcessingIssue
+
+
+class ActionPriority:
+    HIGH = 1
+    MEDIUM = 2
+    LOW = 3
+    UNKNOWN = 4
+
+
+fileNameBlocklist = ["@webkit-masked-url"]
+
+priority_ranking = {
+    EventError.INVALID_DATA: ActionPriority.LOW,
+    EventError.INVALID_ATTRIBUTE: ActionPriority.LOW,
+    EventError.VALUE_TOO_LONG: ActionPriority.LOW,
+    EventError.FUTURE_TIMESTAMP: ActionPriority.LOW,
+    EventError.PAST_TIMESTAMP: ActionPriority.LOW,
+    EventError.CLOCK_DRIFT: ActionPriority.LOW,
+    EventError.INVALID_ENVIRONMENT: ActionPriority.LOW,
+    EventError.SECURITY_VIOLATION: ActionPriority.LOW,
+    EventError.RESTRICTED_IP: ActionPriority.LOW,
+    EventError.FETCH_GENERIC_ERROR: ActionPriority.LOW,
+    EventError.JS_MISSING_SOURCES_CONTENT: ActionPriority.MEDIUM,
+    EventError.JS_SCRAPING_DISABLED: ActionPriority.MEDIUM,
+    EventError.NATIVE_BAD_DSYM: ActionPriority.LOW,
+    EventError.NATIVE_MISSING_OPTIONALLY_BUNDLED_DSYM: ActionPriority.LOW,
+    EventError.NATIVE_MISSING_DSYM: ActionPriority.LOW,
+    EventError.PROGUARD_MISSING_MAPPING: ActionPriority.LOW,
+    EventError.PROGUARD_MISSING_LINENO: ActionPriority.LOW,
+    SourceMapProcessingIssue.MISSING_RELEASE: ActionPriority.HIGH,
+    SourceMapProcessingIssue.MISSING_SOURCEMAPS: ActionPriority.HIGH,
+    SourceMapProcessingIssue.URL_NOT_VALID: ActionPriority.HIGH,
+    SourceMapProcessingIssue.NO_URL_MATCH: ActionPriority.HIGH,
+    SourceMapProcessingIssue.PARTIAL_MATCH: ActionPriority.HIGH,
+    SourceMapProcessingIssue.DIST_MISMATCH: ActionPriority.HIGH,
+    SourceMapProcessingIssue.SOURCEMAP_NOT_FOUND: ActionPriority.HIGH,
+    SourceMapProcessingIssue.DEBUG_ID_NO_SOURCEMAPS: ActionPriority.HIGH,
+}
+
+errors_to_hide = [
+    EventError.JS_MISSING_SOURCE,
+    EventError.JS_INVALID_SOURCEMAP,
+    EventError.JS_INVALID_SOURCEMAP_LOCATION,
+    EventError.JS_TOO_MANY_REMOTE_SOURCES,
+    EventError.JS_INVALID_SOURCE_ENCODING,
+    EventError.UNKNOWN_ERROR,
+    EventError.MISSING_ATTRIBUTE,
+    EventError.NATIVE_NO_CRASHED_THREAD,
+    EventError.NATIVE_INTERNAL_FAILURE,
+    EventError.NATIVE_MISSING_SYSTEM_DSYM,
+    EventError.NATIVE_MISSING_SYMBOL,
+    EventError.NATIVE_SIMULATOR_FRAME,
+    EventError.NATIVE_UNKNOWN_IMAGE,
+    EventError.NATIVE_SYMBOLICATOR_FAILED,
+]
+
+deprecated_event_errors = [
+    EventError.FETCH_TOO_LARGE,
+    EventError.FETCH_INVALID_ENCODING,
+    EventError.FETCH_TIMEOUT,
+    EventError.FETCH_INVALID_HTTP_CODE,
+    EventError.JS_INVALID_CONTENT,
+    EventError.TOO_LARGE_FOR_CACHE,
+    EventError.JS_NO_COLUMN,
+]
+
+
+# These checks mirror what we do in the front end in getUniqueFilesFromException
+def find_debug_frames(event):
```

Comment:
@roggenkemper how do we limit this to just FE JS events?

Reply:
@roggenkemper but aren't we going to call this endpoint on all issues? Not just JS issues?

Reply:
ah blip on my part, will add in a check